configure.ac

```diff
@@ -116,6 +116,30 @@ AX_CXX_COMPILE_STDCXX_11
 
 ######################################################
 
+AC_ARG_ENABLE([repl],
+  [AS_HELP_STRING([--enable-repl],
+    [Enable the repl library @<:@no@:>@])],
+  [repl="$enableval"],
+  [repl="no"])
+AM_CONDITIONAL([REPL], [test x$repl = xyes])
+if test x$repl = xyes; then
```

Comment:
This part (within the if test ...) needs to be moved further down in configure.ac, after `AC_CHECK_LIB([xentoollog], [xtl_logger_destroy])`